src/helpers/getMovieUtils.ts

```diff
@@ -0,0 +1,13 @@
+import { MovieData, MovieDetailsData } from '../types/tmdb';
+
+const getMoviesByName = async (name: string) => {
```

Comment:
JSDoc and return type would be nice here and below. Not blocking but a comment about these queries differ from each other will probably be very helpful down the line. I know the diff but I don't know that it is immediately clear.